ui/blocks/LastCheckin.tsx

```diff
@@ -31,8 +33,15 @@ export const LastCheckin: React.FC<Props> = ({ checkin, area, onCheckout }) => {
       <Text variant="h2">{checkedOut ? 'Checked out' : 'Welcome'}</Text>
       <Box height={1} />
       <Text variant="h4">{checkin.business}</Text>
-      <Box height={4} />
+      <Box height={2} />
       <CheckinDates from={checkin.enteredAt} to={checkin.leftAt} />
+      <Box height={4} />
+      <Text variant="regular">{guestInfo.data?.name}</Text>
+      <Box width={1} />
```

Comment:
What are those `<Box width={1} />` doing? 🤔 Since they boxes don't have a height, they shouldn't do anything here, or do they?

Reply:
Indeed. Forgot to update this.